message_ix/models.py

```diff
@@ -378,17 +378,7 @@ def run(self, scenario):
         lines2 = ("{} = {}".format(*kv) for kv in self.cplex_opts.items())
         optfile2.write_text("\n".join(lines2))
 
-        try:
-            result = super().run(scenario)
-        finally:
-            # Remove the optfile regardless of whether the run completed without error.
-            # The file may have been removed already by another run (in a separate
-            # process) that completed before this one.
-            # py37 compat: check for existence instead of using unlink(missing_ok=True)
-            if optfile.exists():
-                optfile.unlink()
-            if optfile2.exists():
-                optfile2.unlink()
+        result = super().run(scenario)
```

Comment:
Can you add a comment here noting that we previously cleaned up the optfile here but consciously choose not to now because it can cause issues when running scenarios asynchronously